test_scripts/Defects/5_0/2482_UNSUPPORTED_RESOURCE_response_is_received_for_SetGlobalProperties_RPC_when_KEYPRESS_MODE_is_set_to_SINGLE_KEYPRESS_or_QUEUE_KEYPRESSES_or_RESEND_CURRENT_ENTRY.lua

```diff
@@ -0,0 +1,165 @@
+    { type = "TEXT", text = "pathToFile3" }
+  },
+  interactionLayout = "KEYBOARD"
+}
+
+local performHmiParams = {
+  initialPrompt = {
+    { type = "TEXT", text = "pathToFile1"}
+  },
+  helpPrompt = {
+    { type = "TEXT", text = "pathToFile2"}
+  },
+  timeoutPrompt = {
+    { type = "TEXT", text = "pathToFile3"}
+  }
+}
+
+--[[ Local Functions ]]
+local function setGlobalProperties(pKeypressMode)
+  if not pKeypressMode then pKeypressMode = "SINGLE_KEYPRESS" end
+	local cid = common.getMobileSession():SendRPC("SetGlobalProperties", paramsSetGlobalProp)
+
+	responseUiParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("UI.SetGlobalProperties", responseUiParams)
+	:Do(function(_,data)
+		common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	responseTtsParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("TTS.SetGlobalProperties", responseTtsParams)
+	:Do(function(_,data)
+		common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+	common.getMobileSession():ExpectNotification("OnHashChange")
+end
+
+local function createInteractionChoiceSet()
+  local choiceParams = {
+    interactionChoiceSetID = 100,
+    choiceSet = {
+      {
+        choiceID = 111,
+        menuName = "Choice111",
+        vrCommands = { "Choice111" }
+      }
+    }
+  }
+  local corId = common.getMobileSession():SendRPC("CreateInteractionChoiceSet", choiceParams)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+local function sendPerformInteraction_SUCCESS()
+  local corId = common.getMobileSession():SendRPC("PerformInteraction", performParams)
+  common.getHMIConnection():ExpectRequest("UI.PerformInteraction")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getHMIConnection():ExpectRequest("VR.PerformInteraction", {
+    initialPrompt = performHmiParams.initialPrompt,
+    helpPrompt = performHmiParams.helpPrompt,
+    timeoutPrompt = performHmiParams.timeoutPrompt
+  })
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+for _, v in ipairs(keyPressMode) do
+	runner.Step("SetGlobalProperties with keypressMode " .. v, setGlobalProperties, { v })
```

Comment:
```
runner.Step("CreateInteractionChoiceSet", createInteractionChoiceSet)
runner.Step("Send PerformInteraction SUCCESS response", sendPerformInteraction_SUCCESS)
```
these functions also must be executed in the loop for different keypress modes, in current implementation there is no reason of the loop and PI will be performed with RESEND_CURRENT_ENTRY only